docs_app/content/tutorial/7-flattening-operations.md

```diff
@@ -0,0 +1,132 @@
+# 7. Flattening Operations
+
+## TLDR:
+
+- "Flattening" is unwrapping nested (inner) Observables. (`Observable<Observable<T>> -> Observable<T>`)
+- There are four major flattening operations in RxJS: 
+    - **concat**: [`concatMap`](API), [`concatAll`](API), [`concat`](API), and [`concatWith`](API). Flattening one inner observable at a time, in order, without skipping any.
+    - **merge**: [`mergeMap`](API), [`mergeAll`](API), [`merge`](API), and [`mergeWith`](API). Flattening every inner observable, as soon as it can, without skipping any.
+    - **switch**: [`switchMap`](API), [`switchAll`](API). Flattening one inner observable at a time, but unsubscribing from it and "switching" to the newest one as soon as it arrives.
+    - **exhaust**: [`exhaustMap`](API), [`exhaust`](API). Flattening one inner observable at a time, skipping all other inner observables until the one it is subscribed to is completely exhausted (completed).
+- If you're dealing with HTTP, you *probably* wanted [`concatMap`](API) and not [`switchMap`](API).
```

Comment:
wanted -> want.

It probably doesn't belong here, but I would be curious to know why. I must confess I alsays use switchMap to chain http requests with Angular's HttpClient.

Reply:
I would think it would depend on *which* http operations. It makes sense to use switchMap when retrieving data based on user entries/clicks to quickly switch to the newly entered user info. It makes sense to use concatMap when doing insert/update/delete operations as the order of these operations matter. 

Consider updating this TLDR to distinguish between gets and post/put/delete? "If you're dealing with HTTP get, you *probably* want switchMap. For other HTTP operations (put, post, delete) you *probably* want concatMap."